runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.tsx

```diff
@@ -1,9 +1,134 @@
+
+    const [selectedRoles, setSelectedRoles] = useState({});
+    const [selections, setSelections] = useState([]);
+
+    const handleSelectionChange = (selection) => setSelections(selection);
+    const handleSelectChange = (id, value) => setSelectedRoles({ ...selectedRoles, [id]: value});
+
+    const handleAccept = () => {
+        const payload = [];
+        let status = true;
+        for (const selection of selections) {
+            const role = selectedRoles[selection];
+            if (role) {
+                const dataPayload = makePayload(selection, role);
+                payload.push(dataPayload);
+            } else {
+                enqueueSnackbar(translate('Users.Register.View.Events.Error.RolesNotSelected'), { variant: 'error' });
+                status = false;
+                break;
+            }
+        }
+        
+        if (status) {
+            handleSubmit(payload);
+            enqueueSnackbar(translate('Users.Register.View.Events.Success.AcceptingUser'), { variant: 'success' });
+        }
+    };
+    
+    const makePayload = (id: number, role: string) => {
+        const { login } = allNotActivatedByPage.content.find((row) => row.id === id);
+        return { id, login, roles: [role], activated: true };
+    };
+    
+    const handleSubmit = (params: IUser[]) =>
+        params.forEach((param) => dispatch(usersActions.updateNotActivated(param)));
+
+    const handleDelete = () => {
+
+    };
+    
+    useEffect(() => {
+        const pageNotAvailable = allNotActivatedByPage && page >= allNotActivatedByPage.totalPages;
+        if (pageNotAvailable) {
+            router.replace({ pathname: router.pathname, query: { page: 0, pageSize: limit } });
+            setPage(0);
+        }
+        
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [allNotActivatedByPage]);
+
+    useEffect(() => {
+        refreshSearch();
+        
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [page, limit]);
 
     return (
-        <> Register </>
+        <>
+            <StyledActionsContainer>
+                <StyledTextField
+                    margin='dense'
+                    placeholder={translate('Users.Register.View.SearchBarPlaceholder')}
+                    size='small'
+                    value={search}
+                    onChange={handleSearch}
+                />
+                <StyledButtonsContainer>
+                    <StyledButton
+                        type='submit'
+                        variant='contained'
+                        color='primary'
+                        onClick={handleAccept}
+                        disabled={!selections.length}
+                    >
+                        {translate('Users.Register.View.Button.Accept')}
+                    </StyledButton>
+                    <StyledButton
+                        type='submit'
+                        color='error'
```

Comment:
Kamila M. made a request to not create red buttons at all. 

take this 'remove' button idle and hover colors as a source
<img width="954" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/820dda5e-1b1c-4c91-9e00-79304b64be1d">